### DISCUSSION TITLE
Name of `min`, `median`, and `max`

### DISCUSSION THREAD

**Discussion by tedmoore** [flucoma/flucoma-core#122] [Ideas]
This is not just applicable to this repo but is organisation wide. @tremblap @jamesb93 @weefuzzy @jdchart 

Regarding BufStats, is there a reason that we call it "min", "median", and "max" (which is how it's described across all our materials) and yet allow for the user to pass different percentiles (which could end up making it not any of those values)? 

Should we be saying "low percentile", "middle percentile", and "high percentile" which matches what those arguments are called...and then explain how the defaults of 0, 50, and 100 will return the min, median, and max?

It's confusing to say something like `...returning seven statistical properties for each channel: the buffer channel's mean, standard deviation, skewness, kurtosis, minimum, median, and maximum values...` and then have to explain that if the user changes the `low` argument the thing we were calling "min" is now not the minimum.

Any explanations or thoughts or suggestions are appreciated.

**Comment by jamesb93** [comment]
![image](https://user-images.githubusercontent.com/13281862/159525215-cae0e292-0e8f-4349-9a9e-100aed789a25.png)
 because im lazy....

perhaps we should also move this discussion to a more pertinent repo, such as core or docs.
